src/main/resources/com/databricks/spark/sql/perf/mllib/config/mllib-large.yaml

```diff
@@ -67,4 +67,27 @@ benchmarks:
       numItems: 6000000
       regParam: 0.01
       rank: 10
-      maxIter: 10
\ No newline at end of file
+      maxIter: 10
+  - name: regression.DecisionTreeRegression
+    params:
+      depth: [5, 10]
+      numClasses: 4
+  - name: regression.GLMRegression
+    params:
+      numExamples: 500000
+      numTestExamples: 500000
+      numFeatures: 1000
+      link: log
+      family: gaussian
+      tol: 1e-6
```

Comment:
If we're testing for time/iteration, then I'd set tol to 0 and use maxIter to set the iteration count.